src/code42cli/cmds/devices.py

```diff
@@ -103,7 +100,7 @@ def _change_device_name(sdk, guid, name):
 @sdk_options()
 def show(state, device_guid, format=None):
     """Print device info. Requires device GUID."""
-    _DEVICE_INFO_KEYS_MAP = OrderedDict()
+    _DEVICE_INFO_KEYS_MAP = dict()
```

Comment:
We also don't need this syntax, we could just initialize it like `{"name": "Name"...  }`